### PR TITLE
Adds cubic-bezier() and steps() easing functions

### PR DESCRIPTION
**Pull request by vwallen**

### CODE REVIEW

File: features/cubic-bezier-easing.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: cubic-bezier() easing
+description: The `cubic-bezier()` easing function interpolates along a smooth curve, creating animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.
+spec: https://drafts.csswg.org/css-easing-2/#cubic-bezier-easing-functions
+group: css
```
Comment (jamesnw) [inline features/cubic-bezier-easing.yml:4]: Should this go in the `Transitions (CSS)` group?

**Review by jamesnw — CHANGES_REQUESTED** [review]

File: features/cubic-bezier-easing.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+compat_features:
+  - css.types.easing-function.cubic-bezier
```
Comment (ddbeck) [inline features/cubic-bezier-easing.yml:6]: I sent https://github.com/mdn/browser-compat-data/issues/24682 due to all the keyword values not being in BCD.

File: features/cubic-bezier-easing.yml (hunk 1 above)
Comment (vwallen) [reply features/cubic-bezier-easing.yml:4]: Probably both the `Transitions (CSS)` and `Animations (CSS)` groups

Comment (vwallen) [reply features/cubic-bezier-easing.yml]: Updated the groups to include both `transitions` and `animation`, but this is now blocked pending GH-1919, where the `transitions` group will be finalized

**Comment by ddbeck** [comment]
This is blocked on https://github.com/web-platform-dx/web-features/pull/1919. (It's easy to miss in the other comments.)

**Review by jamesnw — APPROVED** [review]
🚀

**Pull request merged**